pkg/cleaner/jobcontroller.go

```diff
@@ -75,10 +74,19 @@ func NewCleanupJob(bd *v1alpha1.BlockDevice, volMode VolumeMode, tolerations []v
 	mountName := "vol-mount"
 
 	if volMode == VolumeModeBlock {
+		jobContainer.Command = []string{"/bin/sh", "-c"}
+
+		// fdisk is used to get all the partitions of the device.
+		// first all the partitions are cleared off any filesystem signatures, then the actual partition table
+		// header is removed. partprobe is called so as to re-read partition table, and update system with
+		// the changes/
 		// wipefs erases the filesystem signature from the block
 		// -a    wipe all magic strings
 		// -f    force erasure
-		jobContainer.Command = getCommand(BlockCleanerCommand, "-af", bd.Spec.Path)
+		args := fmt.Sprintf("(fdisk -o Device -l %s | grep \"^%s\" | sort -r | xargs -I '{}' wipefs -fa '{}') && wipefs -fa %s && partprobe %s",
```

Comment:
we can remove sort  as it is not needed here.

Reply:
Done.